### ISSUE TITLE
Duplicate textBox-elements in PDF (drawBot as module)

### ISSUE THREAD

**Issue by yanone** [typemytype/drawbot#83]
Now that I’ve veryfied active developers over here, please allow me to raise another issue ;)
But here I’m not sure where the problem could lie. It could also be in my own code somewhere, but also maybe not.

I’m suffering from a weird behaviour where text elements (created with both the text() and textBox() commands) appear twice on a PDF’s page, starting after the first textBox() use.
The behaviour restarts on every page. All works fine until the first textBox() call. Thereafter any text elements appears twice excatly on top of each other.

I'm not writing the drawBot-commands directly, but using an exporter for my own graphics abstraction layer library that also has exporters for other systems. But I've verified the singular existence of each text command in several ways, including checking in `_drawBotDrawingTool._instructionsStack` so I'm really confused.

Does that sound familiar to anyone?

**Comment by typemytype** [comment]
call `newDrawing` explicitly to clear the stack.
see http://www.drawbot.com/content/canvas/pages.html#drawBot.newDrawing

**Comment by yanone** [comment]
The script gets called from the command line, so there’s nothing in the stack. I tried `newDrawing()`, but to no avail. And if I call it together with `newPage()` then of course all the previous pages are lost. Also I added `endDrawing()`, but nothing changed.

So I don’t know what to look for. The `_drawBotDrawingTool._instructionsStack` only shows one entry for each item, but the PDF has two (mostly, as described above).
